doc/guides/dts/macros.bnf

```diff
@@ -148,7 +150,8 @@ other-macro =/ %s"DT_CHOSEN_" dt-name
 other-macro =/ %s"DT_COMPAT_" dt-name %s"_BUS_" dt-name
 ;   #define DT_COMPAT_HAS_OKAY_vnd_dev 1
 other-macro =/ %s"DT_COMPAT_HAS_OKAY_" dt-name
-
+; Allows mapping a label to a device node
```

Comment:
Make this `Allows mapping a "label" property to a compatible node.`  Or something that makes clear the label is a property.

Reply:
Done